.github/actions/paths-filter/action.yml

```diff
@@ -20,6 +20,18 @@ outputs:
         steps.filter-common.outputs.java-code-change ||
         steps.filter-common.outputs.maven-change
       }}
+  docker-ci:
+    description:
+    value: >-
+      ${{
+        github.event_name == 'push' ||
+        steps.filter-common.outputs.github-actions-change ||
```

Comment:
Please rebase/update with `main` and notice that these conditions are problematic and all need to be changed to `steps.filter-common.outputs.github-actions-change == 'true'` etc.